llvm-spirv/test/entry_point_func.ll

```diff
@@ -0,0 +1,21 @@
+;; Test to check that an LLVM spir_kernel gets translated into an
```

Comment:
This test looks a bit different in the upstream Khronos Translator. Can you please check?
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/main/test/entry_point_func.ll

Thanks

Reply:
Seems that it was the initial version of the test. That led to the thing that we did not apply the changes from the following commit. Will update the test, thanks for the catch!